### DISCUSSION TITLE
去海灘淨灘

### DISCUSSION THREAD

**Discussion by Jamesmoneya** [1101-GS4719-C/Python-Community-GS4719-C#1436] [General]
上週末，我和一群同學一起去淨灘活動，這是一次很有意義的經驗。我們走在海灘上，一邊欣賞著美麗的海景，一邊尋找著散落在沙灘上的垃圾。雖然一開始我們只是想要做些簡單的環保活動，但當我們發現大量的垃圾時，我們的心情就不自覺地變得嚴肅了起來。
我們分成小組，每個人都拿著一個垃圾袋，開始了清理垃圾的工作。有些垃圾較輕，例如塑膠袋、吸管等，但也有些垃圾非常重，例如船隻的繩索等。我們必須一點一點地將垃圾拾起來，確保不留下任何一個小塊。

在這次活動中，我學到了很多關於環保的知識。我們通過實際的行動來保護我們的地球。這次活動不僅讓我們的環保意識得到了提升，同時也增進了我們彼此之間的友誼。當我們一起清理垃圾時，互相鼓勵和支持，讓我們感到非常溫暖。

我們最終成功地撿起了四公斤的垃圾，這是我們的驕傲。我們從這次活動中得到了很多啟示，並且決心要更積極地參與環保活動，為我們的地球盡一份心力。我們也希望更多的人可以加入到這個行動中來，一起為我們的地球做出貢獻。

**Comment by pigeon9975** [comment]
我上個月也有跟參加系學會辦的淨灘活動，在海灘上撿到了一堆酷東西，像從對岸飄來的垃圾、不知道甚麼動物的骨頭，還有硬梆梆的死河豚(感覺已經變成標本的那種)，在維護環境的同時，我們注意到了好多之前去海邊玩不會注意的小細節，最後檢了大約20公斤的垃圾，雖然很累，但好快樂、好有成就感！

**Comment by alicia0725** [comment]
之前也有參加過類似的活動，淨灘不外乎就是撿到各種人工浪費的製品，若那個區域污染特別嚴重的話，像我還撿到了死掉的一些生物、廢棄漁網等等。這個時候就要反思為何人類種習慣亂丟垃圾，若在海邊設置固定垃圾桶來定時回收會不會有機會可以改善這個問題？

**Comment by ALAN30359704** [comment]
我認為淨灘雖然看似環保，但是我們淨灘的垃圾最終是否又回到了大海呢？然後大海再把垃圾打回海灘上嗎？因為有非常多的垃圾掩埋場就在出海口附近，因為我也曾經在桃園的觀音淨灘過，但是我認為這些垃圾到最後是去哪裡了？我們放到垃圾車，然後垃圾車再把垃圾放到垃圾掩埋場，那這樣我們並沒有解決減少垃圾的問題！

**Comment by 111607004** [comment]
前幾周剛去系上辦的淨灘活動，雖然很熱很累，但真的蠻讚蠻好玩的。真正來到比較沒人去的海灘才會發現現場到底有多髒，一群人把地上的垃圾漸漸淨空，覺得很快樂，很有成就感！

**Comment by pudding0803** [comment]
我沒有淨灘過，但之前就有想試試看，改天有機會會嘗試看看的！

但滿難過的是，好像滿多人都覺得亂丟垃圾不關自己的事，所以才會有那麼多垃圾🥹

😠

**Comment by Brownnnn78** [comment]
對於塑膠垃圾這個議題真的很難有解答，我們製造那麼多無法回收或燒掉的垃圾最終只能掩埋或漂流到大海，但我們買東西的時候依然習以為常的拿了一堆塑膠帶。之前看大陸一些到夜市的吃播，看到他們拿一小塊麵包就要兩個塑膠袋包著，是不是因為他們地大有很多地方可以掩埋？也想到或許他們製造了一堆垃圾然後漂來台灣讓台灣被垃圾佔據就可以統治台灣了（幻想文）。
